common/script/libs/achievements.js

```diff
@@ -0,0 +1,507 @@
+import i18n from '../i18n';
+
+let achievs = {};
+let index;
+
```

Comment:
Is there a good reason to construct these achievements this way instead of enumerating them in the content files?